examples/client/invokeContract.ts

```diff
@@ -0,0 +1,140 @@
+import {
+    AccountAddress,
+    CcdAmount,
+    ContractContext,
+    ContractTraceEvent,
+    createConcordiumClient,
+} from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+import { Buffer } from 'buffer/index.js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn ts-node <path-to-this-file> [options]
+
+  Required:
+    --receive,  -r  The entrypoint (receive function) that shall be invoked
+    --contract, -c  The index of the contract to query info about
+
+  Options
+    --help,      -h  Displays this message
+    --block,     -b  A block to query from, defaults to last final block
+    --amount,    -a  The amount of microCCD to invoke the contract with, defaults to 0
+    --energy,    -n  The maximum amount of energy to allow for execution, defaults to 1000000
+    --invoker,   -i  The address of the invoker, defaults to the zero account address
+    --endpoint,  -e  Specify endpoint of the form "address:port", defaults to localhost:20000
+    --parameter, -p  The serialized parameters that the contract will be invoked with, as a 
+                     hex string. Will default to an empty hex string meaning no parameters
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            receive: {
+                type: 'string',
+                alias: 'r',
+                isRequired: true,
+            },
+            contract: {
+                type: 'number',
+                alias: 'c',
+                isRequired: true,
+            },
+            block: {
+                type: 'string',
+                alias: 'b',
+                default: '', // This defaults to LastFinal
+            },
+            amount: {
+                type: 'number',
+                alias: 'a',
+                default: NaN,
+            },
+            energy: {
+                type: 'number',
+                alias: 'n',
+                default: NaN,
```

Comment:
I don't think `NaN` would make any sense here?

```suggestion
                default: 10000,
```

Reply:
`NaN` behaves the same as `undefined`, so that the `invokeContract` call will choose the default value itself, which is cleaner than an arbitrary value for this example.

It should be `undefined` instead of `NaN` though, so I'll remove the default value entirely here if that works for you.